ml-agents/mlagents/trainers/trainer.py

```diff
@@ -48,50 +43,15 @@ def __init__(
         :str run_id: The identifier of the current run
         :int reward_buff_cap:
         """
-        self.param_keys: List[str] = []
         self.brain_name = brain_name
         self.run_id = run_id
         self.trainer_parameters = trainer_parameters
+        self.is_training = training
         self.summary_path = trainer_parameters["summary_path"]
         self.stats_reporter = StatsReporter(self.summary_path)
-        self.cumulative_returns_since_policy_update: List[float] = []
-        self.is_training = training
         self._reward_buffer: Deque[float] = deque(maxlen=reward_buff_cap)
```

Comment:
Would it make sense to move self._reward_buffer (and reward_buffer property) to RLTrainer? Seems like it could at least be abstract on Trainer.